### PR TITLE
Archive all_react_app_exports.js

### PR DESCRIPTION
**Pull request by MariaAga**

### CODE REVIEW

**Comment by MariaAga** [comment]
Not sure if the placement is correct and if there should be some conditions?

File: .github/workflows/foreman.yml [hunk 1]
```diff
@@ -122,6 +122,11 @@ jobs:
         run: |
           bundle exec rake db:create
           bundle exec rake db:migrate
+      - name: Archive all_react_app_exports
```
Comment (evgeni) [inline .github/workflows/foreman.yml:127]: You can set
```
if: contains(matrix.task, 'compile')
```
as that's the only task where we compile webpack.

**Review by evgeni — COMMENTED** [review]
I'm afraid you'll have to place it *after* the last task (`bundle exec rake ${{ matrix.task }}`) as otherwise webpack isn't compiled yet

File: .github/workflows/foreman.yml (hunk 1 above)
Comment (MariaAga) [reply .github/workflows/foreman.yml:127]: oh great, I thought it would be more complicated, thanks!

**Review by evgeni — APPROVED** [review]

**Pull request merged**